test/index-static-singleton.js

```diff
@@ -0,0 +1,20 @@
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
+
+var indexStatic;
+
+exports.get = function() {
+  if (!indexStatic) {
+    throw new Error("indexStatic has not yet been built.");
+  }
+  return indexStatic;
+};
+
+exports.build = function(done) {
+  this.timeout(10000);
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) return done(err);
+
+    indexStatic = newIndexStatic;
+    done();
+  });
+};
```

Comment:
This singleton basically just makes it easier to generate the `indexStatic` bundle across multiple test suites, so that each test suite doesn't need to regenerate the bundle from scratch--that would really slow down the running of all the tests.